our-team/guide-our-team.md

```diff
@@ -0,0 +1,159 @@
+# Our Team template guide
+
+---
+Thank you for downloading the Our Team template from The Good Docs Project!
+We welcome any kind of feedback on our templates, both positive and negative.
+To suggest improvements, open an issue on our (https://github.com/thegooddocsproject/templates/issues)[Templates repository].

```

Comment:
todo (blocking): Link syntax should be `[Templates repository](https://github.com/thegooddocsproject/templates/issues)`